lib/cadence/connection/thrift.rb

```diff
@@ -337,16 +343,56 @@ def get_search_attributes
         raise NotImplementedError
       end
 
-      def respond_query_task_completed
-        raise NotImplementedError
+      def respond_query_task_completed(namespace:, task_token:, query_result:)
+        query_result_proto = Serializer.serialize(query_result)
+      request = CadenceThrift::RespondQueryTaskCompletedRequest.new(
+          task_token: task_token,
+          namespace: namespace,
+          completed_type: query_result_proto.result_type,
+          query_result: query_result_proto.answer,
+          error_message: query_result_proto.error_message,
```

Comment:
The above named parameters are different for Cadence. You can see the correct ones here: https://github.com/coinbase/cadence-ruby/blob/master/lib/gen/thrift/shared_types.rb#L4174

They are:
 - `taskToken`
 - `completedType`
 - `queryResult`
 - `errorMessage`

(So just remove `namespace` and camel case everything else)